### PR TITLE
web/storagenode: chart y and x axis reworked

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: number of x axis values reduced;
month added to x axis;
added y axis;
added label above y axis;
y axis dashed grid lines added.

Why: SNOboard enhancements

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/storagenode/src/app/components/VChart.vue [hunk 1]
```diff
@@ -53,21 +53,53 @@ export default class VChart extends Vue {
...
                         minRotation: 0,
+                        callback: function(value, index, values): string | undefined {
+                            const valuesLength = values.length;
+                            const firstDateValue = values[0];
+                            const lastDateValue = values[valuesLength - 1];
+                            const isDateValueFirstOrLast = value === firstDateValue || value === lastDateValue;
+                            const middleDateValue = valuesLength / 2;
+                            const isDateValueInMiddleInEvenAmount = index === (middleDateValue - 1);
+                            const isDateValueInMiddleInNotEvenAmount = index === (Math.floor(middleDateValue));
+                            const isAfterEighthDayOfTheMonth = valuesLength > 8 && valuesLength <= 31;
```
Comment (Barterio) [inline web/storagenode/src/app/components/VChart.vue]: This numbers should be moved to const with meaning description.
Avoid using magic numbers

File: web/storagenode/src/app/components/VChart.vue [hunk 2]
```diff
@@ -7,6 +7,34 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
 
 import { ChartData } from '@/app/types/chartData';
 
+class DayShowingConditions {
+    public day: string;
```
Comment (crawter) [inline web/storagenode/src/app/components/VChart.vue]: private

File: web/storagenode/src/app/components/VChart.vue [hunk 3]
```diff
@@ -7,6 +7,34 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
...
+    public daysArray: string[];
```
Comment (crawter) [inline web/storagenode/src/app/components/VChart.vue]: private

File: web/storagenode/src/app/components/VChart.vue [hunk 4]
```diff
@@ -7,6 +7,34 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
...
+    public middleDateValue: number;
```
Comment (crawter) [inline web/storagenode/src/app/components/VChart.vue]: readonly

File: web/storagenode/src/app/components/VChart.vue [hunk 5]
```diff
@@ -7,6 +7,34 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
...
+    public isDateValueFirstOrLast: boolean;
```
Comment (crawter) [inline web/storagenode/src/app/components/VChart.vue]: readonly

File: web/storagenode/src/app/components/VChart.vue [hunk 6]
```diff
@@ -7,6 +7,34 @@ import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
...
+    public isAfterEighthDayOfTheMonth: boolean;
```
Comment (crawter) [inline web/storagenode/src/app/components/VChart.vue]: readonly

**Pull request merged**